channels/qa.m3u

```diff
@@ -19,42 +19,48 @@ https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
 #EXTINF:-1 tvg-id="AlRayyanAlQadeem.qa" tvg-name="Al Rayyan Al Qadeem" tvg-country="QA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/mVwHGbc.png" group-title="Classic",Al Rayyan Al Qadeem (1080p)
 https://svs.itworkscdn.net/alrayyanqadeemlive/alrayyanqadeem.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Aljazeera.qa" tvg-name="Aljazeera" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Aljazeera
+#EXTINF:-1 tvg-id="AlJazeeraArabic.qa" tvg-name="Al Jazeera Arabic" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/DBJTZaS.png" group-title="News",Al Jazeera Arabic (480p)
 http://live-hls-web-aja.getaj.net/AJA/04.m3u8
-#EXTINF:-1 tvg-id="Aljazeera.qa" tvg-name="Aljazeera" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Aljazeera
+#EXTINF:-1 tvg-id="AlJazeeraArabic.qa" tvg-name="Al Jazeera Arabic" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/DBJTZaS.png" group-title="News",Al Jazeera Arabic (1080p)
 https://live-hls-web-aja.getaj.net/AJA/01.m3u8
-#EXTINF:-1 tvg-id="AljazeeraDocumentery.qa" tvg-name="Aljazeera Documentery" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Aljazeera Documentery
+#EXTINF:-1 tvg-id="AlJazeeraDocumentary.qa" tvg-name="Al Jazeera Documentary" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/El2hhYb.png" group-title="Documentary",Al Jazeera Documentary (1080p)
 http://live-hls-web-ajd.getaj.net/AJD/01.m3u8
-#EXTINF:-1 tvg-id="AljazeeraMubasher.qa" tvg-name="Aljazeera Mubasher" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Aljazeera Mubasher
+#EXTINF:-1 tvg-id="Al Jazeera Mubasher.qa" tvg-name="Al Jazeera Mubasher" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/xx6uOhZ.png" group-title="Legislative",Al Jazeera Mubasher (1080p)
 http://live-hls-web-ajm.getaj.net/AJM/01.m3u8
-#EXTINF:-1 tvg-id="beINSports1Asia.qa" tvg-name="beIN Sports 1 Asia" tvg-country="QA" tvg-language="" tvg-logo="https://static.epg.best/qa/beINSports1.qa.png" group-title="",beIN Sports 1 Asia
+#EXTINF:-1 tvg-id="beINSports1Asia.qa" tvg-name="beIN Sports 1 Asia" tvg-country="QA" tvg-language="" tvg-logo="https://static.epg.best/qa/beINSports1.qa.png" group-title="Sports",beIN Sports 1 Asia
 https://liveanevia.mncnow.id/live/eds/BeIN1/sa_dash_vmx/BeIN1.mpd
-#EXTINF:-1 tvg-id="DohatAlQuraan.qa" tvg-name="Dohat Al-Quraan" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",Dohat Al-Quraan
-http://45.92.69.131:8001/play/a01d
+#EXTINF:-1 tvg-id="beINSports1English.qa" tvg-name="beIN Sports 1 English" tvg-country="" tvg-language="English" tvg-logo="https://www.lyngsat.com/logo/tv/bb/bein-sports-english-1-qa.png" group-title="Sports",beIN Sports 1 English
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
+https://www.livedoomovie.com/02_epl1_720p_UK/chunklist.m3u8
```

Comment:
tested in vlc not working

Reply:
Both worked on the time I've tested them, apparently the streams this host has got aren't stable.

Reply:
![image](https://user-images.githubusercontent.com/5430116/121603660-c74ff580-ca51-11eb-9c1f-07b2e2590e53.png)
test properly. user agent is used for a reason